Lombiq.BaseTheme/Middlewares/RemoveBootstrapMiddleware.cs

```diff
@@ -58,18 +49,41 @@ public class RemoveBootstrapMiddleware
         currentSiteTheme?.Id == FeatureIds.BaseTheme ||
         (currentSiteTheme?.Manifest.ModuleInfo as ThemeAttribute)?.BaseTheme == FeatureIds.BaseTheme;
 
-    private static bool IsVersion5(ResourceDefinition definition) =>
-        Version.TryParse(definition.Version, out var version) &&
-        version.Major == 5;
+    private static IEnumerable<(IList<ResourceDefinition> Resources, ResourceDefinition ResourceToDelete)> GetResourcesToClear(
+        IOptions<ResourceManagementOptions> resourceManagementOptions,
+        string resourceType,
+        bool keepNewest)
+    {
+        var resources = resourceManagementOptions
+            .Value
+            .ResourceManifests
+            .SelectWhere(manifest => manifest.GetResources(resourceType)?.GetMaybe("bootstrap"))
+            .SelectMany(resources => resources
+                .Where(definition => Version.TryParse(definition.Version, out var version) && version.Major == 5)
```

Comment:
Reference the version used in `ResourceManagementOptionsConfiguration` so if we upgrade this doesn't need to be adjusted.